### PR TITLE
Use React and Redux

### PR DESCRIPTION
**Pull request by Graycemuthui**
### Description
In this project I achieved the following:
- Created Routes for both the HomePage and the Details Page
- Used Redux
- Styled the components
- Tested all components
- Used proper Gitflow

### Changes made:
- Create a search bar that filters each item
- Added Nelson Sakwa to the Acknowledgements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+
+## Show your support
+
+- Give a ⭐️ if you like this project!
+
+## Acknowledgements
+
+- The React Library.
+- Football Standings API Documentation.
+- Details Backgroung by Martí Pardo from [pexels](https://www.pexels.com/photo/group-of-men-playing-soccer-1657328/)
```
Comment (TedLivist) [inline README.md:70]: - [ ] Kindly include acknowledgment for Nelson Sakwa, the designerr.

File: src/components/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            ))}
+          </div>
+          <Link to="/Leagues">
+            <BiRightArrowCircle className="main-icon" />
+          </Link>
+        </div>
+      </div>
+      <Leagues />
+    </>
+  );
```
Comment (TedLivist) [inline src/components/HomePage.js:48]: - [ ] Your homepage is expected to contain a filtering mechanism, like a search box or something that allows users to filter the contents on the homepage. Kindly ensure your implement that.

  furkan-home

File: src/components/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        <div className="nav-icons">
+          <FaMicrophone />
+          <IoSettingsOutline />
+        </div>
+      </div>
+      <div className="main-container">
+        <p className="text">
+          Total Available &nbsp; Leagues:
+          {leagues.length}
+        </p>
```
Comment (TedLivist) [inline src/components/HomePage.js:32]: - [ ] In keeping with the design I think this text should be moved away from the right side of the page and more toward the center of the half container to make it legible.

  grace-muthui-right

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Graycemuthui,

Good job so far!
✔️ App is properly deployed
✔️ Tests are passing
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/HomePage.js (hunk 2 above)
Comment (rloterh) [reply src/components/HomePage.js]: This issue is well resolved! ✔️

File: src/components/HomePage.js (hunk 3 above)
Comment (rloterh) [reply src/components/HomePage.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Geegee,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- React Router properly implemented
- State management properly implemented
- Integration test properly implemented
- Good ReadMe
- Clean code
- Nice UI design

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Graycemuthui** [comment]
Thank you

**Pull request merged**